examples/agentInteractionsDemo/app.js

```diff
@@ -175,13 +175,88 @@ new Vue({
         const usersApi = new platformClient.UsersApi();
         const qualityApi = new platformClient.QualityApi();
         const conversationsApi = new platformClient.ConversationsApi();
+        const externalContactsApi = new platformClient.ExternalContactsApi();
         Vue.prototype.$usersApi = usersApi;
         Vue.prototype.$qualityApi = qualityApi;
         Vue.prototype.$conversationsApi = conversationsApi;
+        Vue.prototype.$externalContactsApi = externalContactsApi;
 
         let authenticated = false;
         let agentUserId = "";
 
+        function getCustomerParticipant(conv) {
+            let newConv = conv;
+            newConv.customer = newConv.participants.find((part) => {
+                return part.purpose === "customer" || part.purpose === "external";
```

Comment:
should we also verify that the participant has a name prop here?

Reply:
ah, nevermind.  I see the check below now.

But, a good validation of Jake's comment to make the function functional or rename to make the mutation obvious.

Maybe return the customerParticipant or null, pass that into getCustomerName, and then build up the resulting customer object and add it in `getConversationData`?